omod/src/main/webapp/pages/myaccount/changeSecretQuestion.gsp

```diff
@@ -35,6 +35,7 @@ ${ ui.includeFragment("uicommons", "validationMessages")}
             ]) }
             ${ ui.includeFragment("uicommons", "field/text", [
                     id: "secret-question",
+                    initialValue: "$secretQuestion",
```

Comment:
the value is picked from the server and passed into the text field using groovy, it works pretty fine